sdk/keyvault/TROUBLESHOOTING.md

```diff
@@ -129,6 +133,39 @@ The error `message` may also contain the tenant ID (`tid`) and application ID (`
 2. You are authenticated against a Microsoft Account (MSA) in Visual Studio or another credential provider. See
    [above](#operation-not-permitted) for troubleshooting steps.
 
+### Other Authentication Errors
+
+See our [Azure.Identity troubleshooting guide] for general guidance on authentication errors.
+
+#### Multi-tenant Authentication Issues
+
+If an `AuthenticationFailedException` is thrown with a message similar to:
+
+> The current credential is not configured to acquire tokens for tenant
+
+See our [troubleshooting guide for multi-tenant authentication issues](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/identity/Azure.Identity/TROUBLESHOOTING.md#troubleshoot-multi-tenant-authentication-issues).
+Read our [release notes](https://aka.ms/azsdk/blog/multi-tenant-guidance) for more information about this change.
+
+#### Incorrect Challenge Resource
+
+If an `InvalidOperationException` is thrown with a message similar to:
+
+> The challenge resource 'myvault.vault.azure.net' does not match the requested domain. Set DisableChallengeResourceVerification to true in your client options to disable. See https://aka.ms/azsdk/blog/vault-uri for more information.
```

Comment:
```suggestion
> The challenge resource 'myvault.vault.azure.net' does not match the requested domain. Set DisableChallengeResourceVerification to true in your client options to disable. See [https://aka.ms/azsdk/blog/vault-uri](https://aka.ms/azsdk/blog/vault-uri) for more information.
```

Reply:
Hyperlinks are automatically recognized as hyperlinks so this is unnecessary.